### PR TITLE
Improve rate limiter log(#24459)

### PR DESCRIPTION
**Pull request by bigsheeper**
/kind improvement

### CODE REVIEW

File: internal/proxy/multi_rate_limiter.go [hunk 1]
```diff
@@ -314,7 +314,7 @@ func (rl *rateLimiter) registerLimiters(globalLevel bool) {
 		limit := ratelimitutil.Limit(r)
 		burst := r // use rate as burst, because Limiter is with punishment mechanism, burst is insignificant.
 		rl.limiters.GetOrInsert(internalpb.RateType(rt), ratelimitutil.NewLimiter(limit, burst))
-		log.Info("RateLimiter register for rateType",
+		log.RatedInfo(30, "RateLimiter register for rateType",
```
Comment (xiaofan-luan) [inline internal/proxy/multi_rate_limiter.go:317]: Still too much?
We might want to log only if the quota is set and changed in this round

Comment (bigsheeper) [reply internal/proxy/multi_rate_limiter.go]: updated, info&warn log will be printed if the quota is set and changed, and rated debug log will be printed when quota is normal.

**Comment by bigsheeper** [comment]
rerun ut

**Pull request merged**